homeassistant/components/ecowitt_weather/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "ecowitt_weather",
+  "name": "Ecowitt Weather",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/ecowitt_weather",
+  "requirements": ["ecpatestapitest==0.0.8"],
```

Comment:
This library name doesn't sound serious. There's no link on PyPI to a source on github.

Reply:
Sorry, Pypi has not been released yet

Reply:
Excuse me.What do we need to modify?

Reply:
Here's the library on PyPI:
https://pypi.org/project/ecpatestapitest/

1. The name is not serious.
2. There's no link to a github repository so it's not easy to contribute to the code or verify it.

Reply:
Thank you very much.We'll make some modifications and resubmit the application.